src/Spectre/App_Start/ValidateConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Spectre.App_Start
+{
+    using System.Configuration;
+    using Spectre.Service.Configuration;
+
+    /// <summary>
+    /// Provides config validation logic.
+    /// </summary>
+    public class ValidateConfig
```

Comment:
According to convention: *use noun or noun phrases to name a class*. Maybe `ConfigValidator`?